.github/workflows/build-test-lint.yaml

```diff
@@ -21,15 +21,11 @@ jobs:
     with:
       platforms: "linux/amd64"
       worker: false
-  test:
-    needs: build
-    uses: scientist-softserv/actions/.github/workflows/test.yaml@v0.0.3
-    if: github.repository == 'scientist-softserv/harvard-curiosity'
-    with:
-      worker: false
+      # tag: deca90d9
   lint:
     needs: build
-    uses: scientist-softserv/actions/.github/workflows/lint.yaml@v0.0.3
+    uses: scientist-softserv/actions/.github/workflows/lint.yaml@main
     if: github.repository == 'scientist-softserv/harvard-curiosity'
     with:
       worker: false
+      # tag: deca90d9
```

Comment:
Were these "tag" comments added on purpose? If so, why?

Reply:
they are a nice reminder that you can select a tag and skip the build. I just leave them as a placeholder for when I need it next